tf/aws.1/main.tf

```diff
@@ -105,11 +105,36 @@ resource "aws_spot_instance_request" "foundation-node" {
     }
   }
 
+  provisioner "file" {
+    source      = "files/uploadlog.sh"
+    destination = "/home/ec2-user/uploadlog.sh"
+    connection {
+      host        = "${aws_spot_instance_request.foundation-node.public_ip}"
+      type        = "ssh"
+      user        = "ec2-user"
+      private_key = "${file(var.private_key_path)}"
+      agent       = true
+    }
+  }
+
+  provisioner "file" {
+    source      = "files/crontab"
+    destination = "/home/ec2-user/crontab"
+    connection {
+      host        = "${aws_spot_instance_request.foundation-node.public_ip}"
+      type        = "ssh"
+      user        = "ec2-user"
+      private_key = "${file(var.private_key_path)}"
+      agent       = true
+    }
+  }
+
 
   provisioner "remote-exec" {
     inline = [
-      "curl -L https://harmony.one/node2.sh > node.sh",
-      "chmod +x node.sh rclone.sh fast.sh",
+      "curl -LO https://harmony.one/node.sh",
+      "chmod +x node.sh rclone.sh fast.sh uploadlog.sh",
```

Comment:
suggestion: maybe we could organize files in the home directory in the future? Put all .sh files in a script folder, and all .so files in a lib folder.

Reply:
.so files can be distributed to system lib folder.  We need to package it using package files.